public/app/plugins/panel/gauge/panelcfg.gen.ts

```diff
@@ -15,11 +15,13 @@ export interface Options extends common.SingleStatBaseOptions {
   minVizWidth: number;
   showThresholdLabels: boolean;
   showThresholdMarkers: boolean;
+  sizing: common.BarGaugeSizing;
 }
 
 export const defaultOptions: Partial<Options> = {
-  minVizHeight: 75,
-  minVizWidth: 75,
+  minVizHeight: 200,
+  minVizWidth: 200,
```

Comment:
@nmarrs this is a pretty big breaking change, radically changing the look of thousands of panels that used to show many gauges now just show a few and with a scrollbar.

Reply:
You can get it causes issues.. Not cool.